plugins/scaffolder-actions/scaffolder-backend-module-aws/src/actions/ecr/create.ts

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2021 Larder Software Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { createTemplateAction } from '@backstage/plugin-scaffolder-backend';
+import {
+  ECRClient,
+  CreateRepositoryCommand,
+  CreateRepositoryCommandInput,
+  ImageTagMutability,
+  ECRClientConfig,
+} from '@aws-sdk/client-ecr';
+import { CredentialProvider } from '@aws-sdk/types';
+import { assertError } from '@backstage/errors';
+
+export function createEcrAction(options?: {
+  credentials?: CredentialProvider;
+}) {
+  return createTemplateAction<{
+    repoName: string;
+    tags: Array<any>;
+    imageMutability: boolean;
+    region: string;
+    values: any;
```

Comment:
This is not used anmywhere right ?

Reply:
those are the inputs for the action itself, used for the repo creation call

Reply:
those are the inputs for the action itself, used for the repo creation call

Reply:
Ah i wanted to highlight the `values: any` I did a search and didn't found using the values property. The others are used in the handler on the `ctx.input.imageMutability` for exmaple.

Reply:
yes values var isn't needed

Reply:
I've removed the unneeded vars now.